### ISSUE TITLE
disable bird service??

### ISSUE THREAD

**Issue by jerkstore369** [bwesterb/blame-bird#6]
I've never nor do I ever plan on using iCloud or anything icloud related. In High Sierra if you disable com.apple.bird completely, by unloading and disabling it with , macOS now gives some error message about how icloud drive doesn't work properly. No shit Sherf*ck. I don't want it to work properly. Any solutions? Thanks.

**Comment by rubenvarela** [comment]
This doesn't really relate to this repo. 

But to answer your question, you can backup and then delete the file, 

    /System/Library/LaunchAgents/com.apple.bird.plist

or look into `launchctl`

**Comment by jerkstore369** [comment]
Yes, I used Launchctl. It's not a huge deal I guess but it's weird that now it error prompts in HS when in Sierra it didn't. Thanks anyways.
